src/pii-client.ts

```diff
@@ -0,0 +1,79 @@
+import PIIsdk, {
+    AgentType,
+    initAgent,
+    PIIEncryptionMethod,
+} from "@notabene/pii-sdk";
+import { TransactionArguments, TravelRuleOptions } from "./types";
+
+export class PIIEncryption {
+    private readonly config: TravelRuleOptions;
+    public toolset: PIIsdk;
+
+    constructor(config: TravelRuleOptions) {
+        this.config = config;
+        const requiredFields = [
+            "kmsSecretKey",
+            "baseURLPII",
+            "audiencePII",
+            "clientId",
+            "clientSecret",
+            "authURL",
+            "jsonDidKey",
+        ];
+        const missingFields = requiredFields.filter(
+            (field) => !(field in this.config)
+        );
+
+        if (missingFields.length > 0) {
+            throw new Error(
+                `Missing configuration fields: ${missingFields.join(", ")}`
```

Comment:
Please change the error message to `Missing PII configuration fields` (or equivalent) so you user will know where the missing fields are